lib/constants.ts

```diff
@@ -4,6 +4,10 @@ export const SITE_DESCRIPTION =
   "An ecosystem product studio dedicated to global Ethereum ecosystem development and as such building a better Internet."
 export const GEODEWORK_URL = "https://geodework.com"
 export const SITE_URL = process.env.NEXT_PUBLIC_SITE_URL || GEODEWORK_URL
+export const BLOG_NAME = SITE_NAME + " Blog"
+export const BLOG_DESCRIPTION =
+  "Insights, updates, and stories from the Geodework team."
```

Comment:
Quick filler description; of course feel free to update

Reply:
updated: https://github.com/geodework/geodework-website/pull/42/commits/0b7bc04da18a885556df682556809e96f07a45f6